README.md

```diff
@@ -77,6 +77,9 @@ for row in result:
     print(row)
 ```
 
+#### Using a Connection Pool (RECOMMENDED)
```

Comment:
I'd like to see this part first. Ideally the person coming into the "How to use this connector" would receive a recommendation to use a connection pool within the first 2-3 sentences. 

I think this means we'd probably also remove the example without the pool, and show an example using the pool instead.